dsa-23au/java-dsa/interval-scheduling/pom.xml

```diff
@@ -49,8 +49,16 @@
           <version>2.22.1</version>
         </plugin>
         <plugin>
-          <artifactId>maven-jar-plugin</artifactId>
-          <version>3.0.2</version>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-jar-plugin</artifactId>
+            <version>3.2.2</version>
+            <configuration>
+                <archive>
+                    <manifest>
+                        <mainClass>dev.codewithfriends.App</mainClass>
+                    </manifest>
```

Comment:
[Assignment and Co-Creation Week 04](https://github.com/TheEvergreenStateCollege/upper-division-cs/blob/main/dsa-23au/notes/docs/week-04/Assignment-04.md#assignment-and-co-creation-week-04)

Generic comment